managedplugin/hub.go

```diff
@@ -27,13 +27,13 @@ func getHubClient(logger zerolog.Logger, ops HubDownloadOptions) (*cloudquery_ap
 func isDockerPlugin(ctx context.Context, c *cloudquery_api.ClientWithResponses, ops HubDownloadOptions) (bool, error) {
 	p, err := c.GetPluginVersionWithResponse(ctx, ops.PluginTeam, cloudquery_api.PluginKind(ops.PluginKind), ops.PluginName, ops.PluginVersion)
 	if err != nil {
-		return false, fmt.Errorf("failed to get plugin information: %w", err)
+		return false, fmt.Errorf("failed to get %s plugin (name: %s/%s) information: %w", cloudquery_api.PluginKind(ops.PluginKind), ops.PluginTeam, ops.PluginName, err)
```

Comment:
Do we want to add the version too? Maybe `failed to get source plugin cloudquery/aws@v23.0.0`?

Reply:
Done in [f799d1a](https://github.com/cloudquery/plugin-pb-go/pull/323/commits/f799d1a33409b8a7d5cd59194a5bad7d2078d243)